### PR TITLE
Update aarch64 blobs

### PR DESCRIPTION
**Pull request by popegeo**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by andraprs — APPROVED** [review]

File: src/common/mod.rs [hunk 1]
```diff
@@ -568,7 +572,7 @@ mod tests {
 
     fn unset_envvar(varname: &String) {
         let _ = unsafe {
-            libc::unsetenv(varname.as_ptr() as *const i8);
+            libc::unsetenv(varname.as_ptr() as *const c_char);
```
Comment (costisin) [inline src/common/mod.rs:575]: why is this change needed? 

```
pub type c_char = i8;
```
[https://doc.rust-lang.org/std/os/raw/type.c_char.html](https://doc.rust-lang.org/std/os/raw/type.c_char.html)

File: src/lib.rs [hunk 2]
```diff
@@ -101,11 +101,18 @@ pub fn build_from_docker(
             .add_info(vec![output_path, "Open"])
         })?;
 
+    let mut kernel_image_name = "bzImage";
+    if std::env::consts::ARCH.eq("aarch64") {
+        kernel_image_name = "Image";
+    } else if std::env::consts::ARCH.eq("x86_64") {
+        kernel_image_name = "bzImage";
+    }
```
Comment (costisin) [inline src/lib.rs:109]:
```suggestion
    let kernel_image_name = match std::env::consts::ARCH.as_str() {
        "aarch64" => "Image",
        "x86_64" => "bzImage",
    }
```

File: src/common/mod.rs (hunk 1 above)
Comment (popegeo) [reply src/common/mod.rs:575]: If you read the text below:
This type will always be either i8 or u8, as the type is defined as being one byte long.

Comment (popegeo) [reply src/common/mod.rs:575]: https://github.com/rust-lang/rust/issues/79089

File: src/lib.rs (hunk 2 above)
Comment (popegeo) [reply src/lib.rs:109]: that's a fancy suggestion, thanks

File: src/common/mod.rs (hunk 1 above)
Comment (costisin) [reply src/common/mod.rs:575]: I got it!

**Review by andraprs — APPROVED** [review]

**Review by laurstefan — APPROVED** [review]
